controllers/ocsinitialization/ocsinitialization_controller.go

```diff
@@ -602,29 +607,28 @@ func (r *OCSInitializationReconciler) reconcileUXBackendSecret(initialData *ocsv
 	secret.Name = "ux-backend-proxy"
 	secret.Namespace = initialData.Namespace
 
-	_, err = ctrl.CreateOrUpdate(r.ctx, r.Client, secret, func() error {
+	err = r.Client.Get(r.ctx, client.ObjectKeyFromObject(secret), secret)
+	if err != nil && !errors.IsNotFound(err) {
+		return err
+	} else if err == nil {
```

Comment:
So whenever the Get operation suceeds the won't function always return from here without going to the controller removal part below?